src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -150,4 +151,18 @@ public static void addPoints(Entity userEntity, int numPoints, DatastoreService
     }
     datastore.put(userEntity);
   }
+
+  /**
+    Adds 20 points to a user for uploading if it has been more than a day since they last uploaded
+  */
+  public static void addUploadPoints(Entity userEntity, DatastoreService datastore){
+
+    QuizTimingPropertiesUtils utils = new QuizTimingPropertiesUtils();
+    if(userEntity.getProperty("lastAwardedUploadPoints") == null || utils.isTimestampOutdated((long) userEntity.getProperty("lastAwardedUploadPoints"))){
+        addPoints(userEntity, 20, datastore);
```

Comment:
the "20" should be a named constant. That way, if we ever want to change it, all of our tests won't break. If we use a named constant, we can change all hard coded references to 20 with a reference to the constant, and then we only ever need to change the number in one spot if we want to change it in the future.

Reply:
Alright I made a long variable called upload points